src/utils/databaseCreator.py

```diff
@@ -0,0 +1,59 @@
+import csv
+import requests
+import time
+import json
+from xml.etree.ElementTree import fromstring, ElementTree
+
+goodreadsId = []
+
+with open('books.csv') as csvfile:
+  readCSV = csv.reader(csvfile, delimiter=',')
+    
+  for row in readCSV:
+    if row[0] != "book_id":
+      # extract googdreads ids
+      ID = row[1]
+      goodreadsId.append(ID)
+
+lenIds = len(goodreadsId)
+
+with open('googleBooksApi.csv', 'a') as file:
+  writer = csv.writer(file)
+  # This is the structure of the CSV that is generated.
+  # Since I observed that Google Books API behaves quite weird and offers weird answers, I decided to add
+  # in our database the first 5 answers the API offers for each popular book that we had in our dataset,
+  # so when a user searches something, he could actually find books that are actually added in the databse 
+  # and can be ordered.
+  writer.writerow(["goodreads_id", "title", "author", "googlebooks_id1", "googlebooks_id2", "googlebooks_id3", "googlebooks_id4", "googlebooks_id5"])
+  for i in range(lenIds):
+    # goodreads accepts one request per second
+    time.sleep(1)
+    id1 = goodreadsId[i]
+    response = requests.get('https://www.goodreads.com/book/show/' + id1 + '.xml?key=c5WeMxjtRTJEknOl5E21g')
```

Comment:
Can you add a comment describing the response format for goodreads API?

Reply:
Added more comments